### PR TITLE
Fix ignored exceptions for makeBodyVisible

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM

**Comment by jridgewell** [comment]
Wait, I did this on purpose for some reason...

File: src/amp.js [hunk 1]
```diff
@@ -63,19 +63,17 @@ try {
 
       maybeValidate(window);
       makeBodyVisible(document, waitForExtensions(window));
-    } catch (e) {
-      makeBodyVisible(document);
     } finally {
+      makeBodyVisible(document);
```
Comment (jridgewell) [inline src/amp.js]: This will immediately make the body visible instead of waiting for the Dynamic CSS extension to load.

File: src/amp.js [hunk 2]
```diff
@@ -63,19 +63,17 @@ try {
 
       maybeValidate(window);
       makeBodyVisible(document, waitForExtensions(window));
-    } catch (e) {
-      makeBodyVisible(document);
```
Comment (jridgewell) [inline src/amp.js]: Re: https://github.com/ampproject/amphtml/pull/2146/files#r53512967, we should keep the `catch` block and just re-throw.

File: src/amp.js [hunk 3]
```diff
@@ -63,19 +63,17 @@ try {
...
     } finally {
+      makeBodyVisible(document);
       perf.tick('e_is');
       // TODO(erwinm): move invocation of the `flush` method when we have the
       // new ticks in place to batch the ticks properly.
       perf.flush();
     }
   }, /* opt_isRuntimeCss */ true);
-} catch (e) {
```
Comment (cramforce) [inline src/amp.js]: Revert this change. We overthought it. It should not always make it visible syncronously.

**Comment by cramforce** [comment]
Since this is urgent I made #2149
